internal/featuresupport/features.go

```diff
@@ -238,19 +296,34 @@ func (feature *LvmFeature) GetSupportLevel(filters SupportLevelFilters) models.S
 	return models.SupportLevelSupported
 }
 
-// DualStackNetworkingFeature
-type DualStackNetworkingFeature struct{}
+func (feature *LvmFeature) IsFeatureActivated(cluster common.Cluster, updateParams *models.V2ClusterUpdateParams) bool {
+	if updateParams != nil && updateParams.OlmOperators != nil {
+		for _, operatorParams := range updateParams.OlmOperators {
+			if operatorParams.Name == "lvm" {
+				return true
+			}
+		}
+	}
 
-func (feature *DualStackNetworkingFeature) GetId() models.FeatureSupportLevelID {
-	return models.FeatureSupportLevelIDDUALSTACKNETWORKING
+	if cluster.MonitoredOperators != nil {
```

Comment:
why do we need to check also  cluster.MonitoredOperators? why updateParams.OlmOperators is not enough?

Reply:
Because that there is few ways to determine if operator is activated or not.
If the user updates the cluster but not updating the (already selected) operators `updateParams` are not enough for that